asmcli/commands/install.sh

```diff
@@ -74,21 +74,19 @@ install_in_cluster_control_plane() {
   fi
 
   local PARAMS
-  PARAMS="-f ${OPERATOR_MANIFEST}"
-  for yaml_file in $(context_list "istioctlFiles"); do
-    PARAMS="${PARAMS} -f ${yaml_file}"
-  done
+  init_install_params
 
   if [[ "${_CI_NO_REVISION}" -ne 1 ]]; then
     PARAMS="${PARAMS} --set revision=${REVISION_LABEL}"
   fi
 
-  if [[ "${K8S_MINOR}" -eq 15 ]]; then
-    PARAMS="${PARAMS} -f ${BETA_CRD_MANIFEST}"
-  fi
-  PARAMS="${PARAMS} -c ${KUBECONFIG}"
   PARAMS="${PARAMS} --skip-confirmation"
 
+  print_config >| "${RAW_YAML}"
+  istioctl manifest generate \
+    <"${RAW_YAML}" \
+    >|"${EXPANDED_YAML}"
+
```

Comment:
Why is this moved before installation?

Reply:
it was for debugging later i found another way to accomplish it so will move it back down.